tests/csvwriter.py

```diff
@@ -5,66 +5,79 @@
 from input_output.file_output import create_csv_file
 import time
 import csv
+from input_output.sys_output import fail, passed
 
 # This file is to demonstrate the workings of writing test results to a CSV file.
 # The test itself is a mock and will always pass.
 # The result of the test are written to CSV in the folder output_files/csv/
-from input_output.sys_output import fail, passed
 
-file = 'colorref_smallexample_4_16'
-filename = 'test_graphs/color_refinement/' + file + '.grl'
-graphs = load_graph_list(filename)
 
-# Output csv creation
-csv_filepath = create_csv_file('csvwriter_test')
-# Write first row with column names
-csv_write_array = [['file', 'i', 'time (s)', 'result']]
+def unit_test():
+    print('<csvwriter>')
```

Comment:
Misschien kun je deze regel ook nog zo maken als in de andere testen, met zo'n linkje naar het bestand en met 'Appending to CSV' enzo.

Reply:
staat erbij, heeft nu hetzelfde gedrag als de andere files